package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@funboxteam/crafter",
-  "version": "4.1.1",
+  "version": "4.1.2",
```

Comment:
We need to bump package version in order to obtain successful "npm publish" pipeline which is a trigger to build and push docker image.